browser/src/TeamPage/__snapshots__/TeamPage.spec.tsx.snap

```diff
@@ -920,7 +920,7 @@ exports[`Team Page has no unexpected changes 1`] = `
               aria-labelledby="website-staff"
               className="c15"
               dangerouslySetInnerHTML={
-                Object {
+                {
                   "__html": "- **Matthew Solomonson**
 - Phil Darnowski
 - Riley Grant
```

Comment:
More general, but there will be some merge conflicts when rebasing this off main. 

I have that dang Downloads page PR still open (I was hoping to wait for the additional dataset to be ready to add that in) that may also cause merge conflicts with this updating `jest` and its snapshots commit here. I could wait on that if this is close to going to prod, otherwise, there may be a bit of rebasing in your future, just fyi.

Reply:
Thanks for the heads-up but the chance (or even likelihood) of a merge conflict isn't a big problem here, definitely not grounds to hold back your branch from `main` if it's ready. In a simple case like this, when you rebase and get the merge conflict, it's generally pretty clear what happened and how to fix it, so small conflicts like this are just speedbumps.

That just goes for small merge conflicts, though. Before git caught on, probably the most popular version control system was Subversion, and its manual had something about this that has stuck with me. They said something to the effect that a good merge algorithm is not as important as communication. That is, if you're frequently having trouble resolving large merge conflicts, you don't have a code problem that you fix with software, you have a team and process problem that you fix with communication.

Reply:
> I was hoping to wait for the additional dataset to be ready to add that in

Why do we need to wait? Couldn't you add it after the fact?